### PR TITLE
feat: add ens username and avatar support

### PR DESCRIPTION
**Pull request by bigint**
Screenshot 2022-01-26 at 11 28 33 AM

### CODE REVIEW

**Comment by bigint** [comment]
cc @xvi10 @gdev8317

**Comment by xvi10** [comment]
@bigint thank you very much for this, will review and merge it soon

File: src/Helpers.js [hunk 1]
```diff
@@ -3,6 +3,7 @@ import { InjectedConnector, UserRejectedRequestError as UserRejectedRequestError
 import { WalletConnectConnector, UserRejectedRequestError as UserRejectedRequestErrorWalletConnect } from '@web3-react/walletconnect-connector'
 import { toast } from 'react-toastify'
 import { useWeb3React, UnsupportedChainIdError } from '@web3-react/core'
+import { getDefaultProvider } from "@ethersproject/providers";
```
Comment (gdev8317) [inline src/Helpers.js:6]: there is `ethers` imported already, i guess `ethers.getDefaultProvider` is the same thing?

File: src/App.js [hunk 2]
```diff
@@ -234,7 +237,10 @@ function AppHeaderUser({
         <NavLink activeClassName="active" className="default-btn" to="/trade">Trade</NavLink>
       </div>
       <a href={accountUrl} target="_blank" rel="noopener noreferrer" className="App-cta small transparent App-header-user-account">
-        {shortenAddress(account, small ? 11 : 13)}
+        <div className="App-header-user-davatar">
+          <Davatar size={20} address={account} />
+        </div>
+        <div>{ensName || shortenAddress(account, small ? 11 : 13)}</div>
```
Comment (gdev8317) [inline src/App.js:243]: Could you please check vertical alignment? Looks like with with ens name it's not centered

Screenshot 2022-01-27 at 14 49 41

Screenshot 2022-01-27 at 14 49 15

**Comment by gdev8317** [comment]
Hi @bigint! Thank you very much for the PR. I've checked it and left 2 comments, could you please take a look?

File: src/App.js (hunk 2 above)
Comment (bigint) [reply src/App.js]: Screenshot 2022-01-27 at 5 41 35 PM

@gdev8317 this is fixed now

**Comment by bigint** [comment]
@gdev8317 resolved your comments 🤗

**Pull request merged**